Applications/Hermes/hermesflaskserver.py

```diff
@@ -0,0 +1,177 @@
+import os
+import hermesobject as hermesobject
+import ConfigParser
+import json
+import base64
+import cPickle
+
+from flask import Flask
+from flask import request
+
+# Open configuration INI
+config = ConfigParser.RawConfigParser()
+filename = os.path.abspath("hermes.ini")
+config.read(filename)
+
+# Global variables
+dictmsgobj = {}
+
+
+def configparse():
+    """ Parses configuration file for callsigns and node ID and returns them in JSON format"""
+    local = {}
+    num = int(config.get('PROXY', 'UNITS'))
+    units = range(0, num)
+
+    for unit in units:
+        # TODO We don't really check for valid input here yet
+        item = "UNIT" + str(unit)
+        callsign = config.get(item, "callsign").upper()
+        nodeid = config.get(item, "nodeid")
+
+        local[str(item)] = \
+            {
+                "callsign": callsign,
+                "nodeid": nodeid,
+            }
+
+    local = json.dumps(local)
+    return json.loads(local)
+
+
+# Initialize Flask micro-framework
+app = Flask(__name__)
+
+
+@app.route('/', methods=['GET', 'POST'])
+def message():
+    """
+    Flask function that provides the "transmit" and "receive" functionality.
+
+    POST:
+
+    Transmits a provided payload data from a specified local unit to a specific remote unit.
+
+    :param localcallsign: Flask parameter - Local unit callsign to transmit from
+    :param localnodeid: Flask parameter - Local unit node ID to transmit from
+    :param destinationcallsign: Flask parameter - Remote unit callsign to transmit to
+    :param destinationnodeid: Flask parameter - Remote unit node ID to transmit to
+    :param data: Flask parameter: Data payload to transmit to remote unit
+
+    :return: Returns JSON success status for Flask server after transmission
```

Comment:
Updated:

` :return: Returns HTTP status along with JSON element containing the next received data from the receiver queue`